tsconfig.json

```diff
@@ -1,82 +1,7 @@
 {
-  "extends": "@tsconfig/svelte/tsconfig.json",
-  "include": ["src/**/*.d.ts", "src/**/*.ts", "src/**/*.js", "src/**/*.svelte"],
-  "references": [{ "path": "./tsconfig.node.json" }],
```

Comment:
I've used (Project References)[https://www.typescriptlang.org/docs/handbook/project-references.html] a lot, but here I elected to configure everything as independent typescript projects, instead.

- Project references are hard to get right
- They don't always play nicely with ESLint
- They can make it difficult to do things like importing a `package.json` file

They could still be helpful here, though! I think a potential future improvement here would be:

1. Move this `tsconfig.json` to `src`
2. Add a top level `tsconfig.json` that uses project references to build the `src`, `tests`, `playground`, and `tsconfig.node.json` projects
3. Mark `src` as a dependency of `playground`

I think a change like that, though, deserves its own PR